habitat_sim/utils/datasets_download.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import os
+
+# TODO: is there a better way to do this more generaly? Maybe pybind a cmake compiled DATA_DIR?
```

Comment:
We probably want to let users specify a path and have the default path just be relative to their current working directory.  Doing relative to this file won't work for the conda install

Reply:
See the notebooks, we already have a very general way that gets the path of the current GitHook.

Reply:
Added: `--data_path` default from git root.